packages/mui-material/src/styles/createThemeWithVars.js

```diff
@@ -176,7 +176,7 @@ export default function extendTheme(options = {}, ...args) {
 
   if (!defaultScheme) {
     throw new MuiError(
-      'MUI: The provided `colorSchemes.%s` to the `extendTheme` function is either missing or invalid.',
+      'MUI: The provided `colorSchemes.%s` is either missing or invalid.',
```

Comment:
I think this reads better:

```suggestion
      'MUI: The `colorSchemes.%s` option is either missing or invalid.',
```